src/facet_display/FacetDisplay.test.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react"
+import { shallow } from "enzyme"
+
+import {
+  default as FacetDisplay,
+  getDepartmentName,
+  getLevelName
+} from "./FacetDisplay"
+import { FacetManifest, Facets } from "./types"
+
+describe("FacetDisplay component", () => {
+  const facetMap: FacetManifest = [
+    {
+      name:               "topic",
+      title:              "Topics",
+      useFilterableFacet: false,
+      expandedOnLoad:     false
+    },
+    {
+      name:               "resource_type",
+      title:              "Types",
+      useFilterableFacet: false,
+      expandedOnLoad:     false
+    },
+    {
+      name:               "department",
+      title:              "Departments",
+      useFilterableFacet: false,
+      expandedOnLoad:     true,
+      labelFunction:      getDepartmentName
```

Comment:
Probably shouldn't decide here...but, do you think we should move the labeling into search API? So bucket results would be `{ key: "whatever", "doc_count": 123, "label": "What Ever Label" }`?